README.md

```diff
@@ -101,7 +101,11 @@ protoc \
   example.proto
 ```
 
-All messages generated include the new `Validate() error` method. PGV requires no additional runtime dependencies from the existing generated code.
+All messages generated include the following methods:
+- `Validate() error` which returns the first error encountered during validation
```

Comment:
for consistency, either add a period at the end of this line or remove the period after the next line